src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -58,6 +58,9 @@ import {
 
 let uniqueId = 0;
 
+/**
+ * Provides an element to attach a single local file
```

Comment:
```suggestion
 * Provides an element to attach a single local file.
```

Reply:
Fixed.

Reply:
Looks like this suggestion wasn't actually committed. Mind if I go ahead and commit it?

Reply:
I hadn't pushed the change. It is there now